app/controllers/comments_controller.rb

```diff
@@ -26,11 +26,13 @@ def show
 
   def like
     CommentService.like(comment: @comment, actor: current_user)
+    @comment.reload
```

Comment:
Need this to pick up the changes to `liker_ids_and_names`, a thing we don't use on the angular side. I will look forward to not needing it anymore.